README.md

```diff
@@ -4,6 +4,8 @@
 
 **An experimental Rust architecture for reactive UI**
 
+[![Crate](https://img.shields.io/crates/v/xilem?style=flat-square)](https://crates.io/crates/xilem)
+[![Documentation](https://img.shields.io/badge/docs-latest-blue.svg?style=flat-square)](https://docs.rs/xilem)
```

Comment:
There are existing badges for these at the bottom of the list of badges, which are commented out.

Could you please change this PR to uncomment those?

Reply:
Oh wow I didn't even notice😅 sounds good I just switched over